src/main/java/at/hannibal2/skyhanni/data/PetAPI.kt

```diff
@@ -35,31 +178,492 @@ object PetAPI {
         "➡",
     )
 
-    fun isPetMenu(inventoryTitle: String): Boolean = petMenuPattern.matches(inventoryTitle)
+    fun isPetMenu(inventoryTitle: String, inventoryItems: Map<Int, ItemStack>): Boolean {
+        if (!petMenuPattern.matches(inventoryTitle)) return false
+
+        val goBackLore = inventoryItems[48]?.getLore().orEmpty()
+        return !goBackLore.any { forgeBackMenuPattern.matches(it) }
```

Comment:
what is this checking exactly? if in a forge menu? please use comments to indicate or more and descriptive variable names

Reply:
![image](https://github.com/user-attachments/assets/35f3bacc-46bb-4aad-a9da-a2c0580b372c)

It's to prevent it happening here